### PR TITLE
Add Polish localization - Docs main index

### PR DESCRIPTION
**Pull request by mfilocha**
This is a part of the initial Polish localization - see: https://github.com/kubernetes/website/pull/18419

### CODE REVIEW

File: content/pl/docs/sitemap.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  padding-top: 20px;
+}
+</style>
+
+Click tags or use the drop downs to filter. Click table headers to sort or reverse sort.
+
+<p id="filters">
+Filter by Concept: <span id="conceptFilter" /><br/>
+Filter by Object: <span id="objectFilter" /><br/>
+Filter by Command: <span id="commandFilter" />
```
Comment (sftim) [inline content/pl/docs/sitemap.md]: nit: I think these are not localized

File: content/pl/docs/sitemap.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+#filters select{
+  font-size: 14px;
+  border: 1px #000 solid;
+}
+#filters {
+  padding-top: 20px;
+}
+</style>
+
+Click tags or use the drop downs to filter. Click table headers to sort or reverse sort.
```
Comment (sftim) [inline content/pl/docs/sitemap.md]: nit: I think this should be translated.

File: content/pl/docs/sitemap.md (hunk 1 above)
Comment (mfilocha) [reply content/pl/docs/sitemap.md]: Localized.

File: content/pl/docs/sitemap.md (hunk 2 above)
Comment (mfilocha) [reply content/pl/docs/sitemap.md]: Translated.

**Comment by sftim** [comment]
/hold
pending approval & merge for #18419. Once that merges, it's OK to /unhold
People can /lgtm (for localization review) and /approve, even whilst this is held.

**Comment by sftim** [comment]
@kpucynski would you be happy to add a /lgtm ?

**Comment by rlenferink** [comment]
/unassign @jaredbhatti 
(this will be taken care of by the Polish localization team once the initial PR merges)

**Comment by sftim** [comment]
Thanks @mfilocha & @nvtkaszpir 
/approve
(Once the initial localization lands, there will be a set of approvers for Polish who can self-manage the localization)

**Comment by rlenferink** [comment]
_Approved based on @sftim being an [sig-docs approver](https://github.com/kubernetes/website/commit/756e5976846d7a112a833ffe7e448eb734ce17d6) (however, the `dev-1.17-pl.1` branch is a bit behind master)_

**Pull request merged**